src/components/structures/LeftPanel.tsx

```diff
@@ -309,6 +309,22 @@ export default class LeftPanel extends React.Component<IProps, IState> {
                 }
                 break;
         }
+        const navAction = getKeyBindingsManager().getNavigationAction(ev);
+        switch (navAction) {
+            case KeyBindingAction.NextLandmark:
+                ev.stopPropagation();
+                ev.preventDefault();
+                (
+                    document.querySelector(".mx_RoomTile_selected") ||
+                    document.querySelector(".mx_RoomListHeader_contextMenuButton")
+                )?.focus();
```

Comment:
```suggestion
                (
                    document.querySelector<HTMLElement>(".mx_RoomTile_selected") ??
                    document.querySelector<HTMLElement>(".mx_RoomListHeader_contextMenuButton")
                )?.focus();
```

Or maybe would the following work

```suggestion
                document.querySelector<HTMLElement>(".mx_RoomTile_selected, .mx_RoomListHeader_contextMenuButton"?.focus();
```

A comment would go a long way too so it can be maintained appropriately, why these two classes